lesson-01-cw721/exercise-05/exercise.md

```diff
@@ -7,11 +7,11 @@ Title: When to use `cw721-base`
 Filename: lib.rs
 -->
 
-> Your head is spinning with new NFT knowledge. Unfortunately, so far, none of it seems helpful for escaping from SECTION 31's passport-control holding cell.
+> Your head is spinning with new NFT knowledge. Unfortunately, so far, none of it seems helpful in escaping from SECTION 31's passport-control holding cell.
 
-In the last two exercises we've been talking about a "token collection contract", which we've been referring to as `some_token`. What is this contract and how does it relate to `cw721` and `cw721-base`?
+In the last two exercises, we have discussed a "token collection contract", which we've been referring to as `some_token`. What is this contract and how does it relate to `cw721` and `cw721-base`?
```

Comment:
"we have discussed" seems a bit formal, maybe we can just use "we've discussed"?